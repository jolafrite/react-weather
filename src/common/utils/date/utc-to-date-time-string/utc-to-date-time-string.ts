export const utcToDateTimeString = (utcDate: number) => {
  const date = new Date(utcDate * 1000);
  return date.toISOString().substr(0, 19).replace('T', ' ');
}
