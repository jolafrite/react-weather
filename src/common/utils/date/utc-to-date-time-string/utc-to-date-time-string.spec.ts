import { utcToDateTimeString } from "./utc-to-date-time-string";

describe('utcToDateTimeString', () => {

  test('convert a utc date to date time string', () => {
    expect(
      utcToDateTimeString(1560416400)
    ).toMatchSnapshot();
  });

  test('convert an other utc date to date time string', () => {
    expect(
      utcToDateTimeString(1560837600)
    ).toMatchSnapshot();
  });

});
