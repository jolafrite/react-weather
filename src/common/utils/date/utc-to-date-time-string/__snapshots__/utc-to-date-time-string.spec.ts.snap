// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utcToDateTimeString convert a utc date to date time string 1`] = `"2019-06-13 09:00:00"`;

exports[`utcToDateTimeString convert an other utc date to date time string 1`] = `"2019-06-18 06:00:00"`;
